### ISSUE TITLE
[Crash] [ARCropolis version]:Game crashing

### ISSUE THREAD

**Issue by Sybenz7** [Raytwo/ARCropolis#286]
### Have you read the disclaimer before proceeding?

- [X] I have read the disclaimer.

### System Firmware Version

14.1.0.

### Atmosphere Version

1.3.1.

### Smash Version

13.0.0

### ARCropolis Version

v3.2.0

### Version channel

Beta

### SD card formatting

exFat

### Can the crash be consistently reproduced?

Yes

### At what moment does the crash happen?

- [ ] Before the Switch boot animation
- [X] After the Switch boot animation
- [ ] During the Smash ball loading on boot
- [ ] Before the opening cutscene, but only sometimes
- [ ] On the Stage Select Screen
- [ ] On the Character Select Screen
- [ ] During the loading for a battle
- [ ] At random
- [ ] After a long time playing

### Context

After trying to use smash after the recent Arcropolis update, the game wont even open

### Crash report

[01649814148_01006a800016e000.log](https://github.com/Raytwo/ARCropolis/files/8477697/01649814148_01006a800016e000.log)

**Comment by Raytwo** [comment]
Sounds to me like the update ended up being broken. Something looks terribly wrong here.
Please download the latest update from github and manually install it, then let us know how it want.

**Comment by maxface14** [comment]
Looks like the issue ended up being Skyline, which kept getting corrupted somehow whenever I tried to transfer it over FTP. The file sizes didn't match. Once I transferred it to the SD card over the wire via my Mac, the game was finally able to run again. Then it was a matter of figuring out which plugins broke following the Skyline/ARCropolis update: for me it was libnn_hid_hook.nro, libparam_hook.nro, and libtraining_modpack.nro, all of which are dependencies of Jugeeya's training modpack. libresults_screen.nro broke as well.

I'm not sure why so many people are experiencing an issue identical to mine when mine seems to have been caused by a freak bug in my FTP client, but it is what it is. I've seen identical complaints here as well as on the Smash Ultimate Modding discord and so far it seems that I'm the only one that's been able to solve the problem. I wish I could help by sending some crash reports or something but it was crashing before the system even realized it should be writing all this down, so no logs were generated. Feel free to dm me at maxface#8199 on Discord if you'd like more information.

I would definitely check out the conflicts with the training modpack. I don't know whether the problem is on your end or Jugeeya's, but it's definitely broken at the moment and I'm imagining that will cause a stir. To everyone else who's experiencing crashes, make sure that the file size on subsdk9 is the same on your SD card as it is on the file you downloaded from github.

**Comment by Raytwo** [comment]
Yeah sadly, FTP is slightly unreliable on Switch (except if you use Filezilla) and it's always better to at least install the plugins through a computer.

We can't really offer any form of debugging for Skyline, as it happens before any plugin even runs, but I'll make sure to keep it in mind to mention in the reworked [Troubleshooting](https://github.com/Raytwo/ARCropolis/wiki/Common-Issues-and-How-To-Fix-Them) page.
